internal/engine/upper_test.go

```diff
@@ -230,6 +199,7 @@ func TestUpper_UpWatchCoalescedFileChanges(t *testing.T) {
 	for _, fileRelPath := range fileRelPaths {
 		f.fsWatcher.events <- watch.FileEvent{Path: fileRelPath}
 	}
+	time.Sleep(time.Millisecond)
```

Comment:
?

this seems like it means you introduced a bug

Reply:
Yeah what we really need is a bit on the state that says "we have successfully watched the watches." Right now there's a slight delay between when we tell Tilt to start the watches and when they actually start.

I'll make a story for this.